packages/charts/src/charts/bar/utils/mark.utils.ts

```diff
@@ -35,22 +36,24 @@ export const calculateMarkPosition = ({
   y,
   width,
   height,
+  positiveValue,
```

Comment:
Instead of `positiveValue` I rather suggest to couple positioning logic with position in cartesian grid. If value is negative then `Y` will be in one of two quadrants that represents negative values.

`inPositiveCartesianQuadrant` - IMO will be better explain this connection between bar value and mark position.

What do You think ?

Reply:
I agree that `inPositiveCartesianQuadrant` is better name 👍